src/store/opportunities/actions.ts

```diff
@@ -10,4 +11,19 @@ const loadOpportunities = createAsyncThunk(
   }
 );
 
-export { loadOpportunities };
+const loadExpandedOpportunity = createAsyncThunk<
+  IFullOpportunity | undefined,
+  string
+>(ActionTypes.LOAD_EXPANDED_OPPORTUNITY, (id: string) => {
+  return opportunitiesApi.getOpportunityById(id);
+});
+
+const unloadExpandedOpportunity = createAction(
+  ActionTypes.UNLOAD_EXPANDED_OPPORTUNITY
+);
```

Comment:
I think it's better to name it via reset
```suggestion
const resetExpandedOpportunity = createAction(
  ActionTypes.RESET_EXPANDED_OPPORTUNITY
);
```